packages/docs/integrate/custom-integration.md

```diff
@@ -0,0 +1,334 @@
+# Creating a simple custom integration
+
+::: warning Want to build an integration?
+If you want to integrate with Vue Storefront, don't hesitate to get in touch with the core team on our [Discord](https://discord.vuestorefront.io/) server. We are eager to help you to ensure its high quality and maybe even officially recommend it 😉
+:::
+
+Vue Storefront provides out-of-the-box integrations for various third-party solutions (you can find them listed [here](https://docs.vuestorefront.io/v2/integrations/)). However, sometimes your project might need to connect to a service that Vue Storefront does not integrate with by default.
+
+For this case, you can leverage our default pattern to create your own integration easily and keep your project coherent!
+
+This quick tutorial will guide you step-by-step through creating a simple integration with [JSON Placeholder Free Fake API.](https://jsonplaceholder.typicode.com/)
+
+## Locating custom integration code
+
+Integrations delivered by Vue Storefront are installed in your project as dependencies. When it comes to creating your own integrations, the easiest solution would be to create a dedicated `/integrations` directory in the root of your project.
+
+To kick off working with this tutorial, please, add the following directory structure to the root of your project (highlighted in blue):
+
+![Untitled](../images/custom-integration-file-structure.png)
```

Comment:
`Untitled` is not a proper image description. Accessibility matters in documentation 😊